dbld/packages.manifest

```diff
@@ -42,16 +42,19 @@ which				[fedora]
 #############################################################################
 # eBPF related tools
 #############################################################################
-clang				[tarball]
-libbpf-dev			[tarball]
+clang				[tarball, devshell]
+libbpf-dev			[tarball, devshell]
 bpftool				[tarball]
```

Comment:
I think `bpftool` should be installed in `devshell` as well, what do you think?

Reply:
It is installed, but on ubuntu it comes from a different package (linux-tools-common and the versioned linux-tools builds)